src/Worker/Notifier.php

```diff
@@ -802,10 +803,15 @@ private static function activityPubDelivery($cmd, array $target_item, array $par
 			// Remote items are transmitted via the personal inboxes.
 			// Doing so ensures that the dedicated receiver will get the message.
 			$inboxes = ActivityPub\Transmitter::fetchTargetInboxes($parent, $uid, true, $target_item['id']);
+
+			if (in_array($target_item['private'], [Item::PUBLIC])) {
+				$relay_inboxes = ActivityPub\Transmitter::addRelayServerInboxes([]);
```

Comment:
You can remove the empty array parameter since it is now the default value.

Reply:
Done